frontend_tests/node_tests/hotkey.js

```diff
@@ -278,6 +279,12 @@ function stubbing(func_name_to_stub, test_function) {
     assert_mapping('left_arrow', 'lightbox.prev');
     assert_mapping('right_arrow', 'lightbox.next');
 
+    hotkey.is_subs = return_true;
+    set_global('ui_state', {home_tab_obscured: return_true});
```

Comment:
@timabbott I thought that you could use `global.ui_state.home_tab_obscured` instead of setting global multiple times. Maybe I'm wrong?